mautic/mautic
# Refactor email clone method

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | yes
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
Refactor  code for email clone method.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create an email
3. Clone above created email
4. Cloning should be work as earlier.
